README.rst

```diff
@@ -552,6 +552,30 @@ otherwise it is relative to the item it is defined in ("Foo" in the above
 example).
 
 
+Path Lookup
+~~~~~~~~~~~
+
+Sometimes you need to resolve an already created object by its path.
+The resolve-path section helps you here:
+
+.. code:: javascript
+
+
+      [
+          {
+              "_path": "foo",
+              "_type": "MyType",
+              "title": "Foo",
+              "relations": "resolvePath::/bar"
+          }
+      ]
+
+Using the ``resolvePath::path`` syntax the value is replaced with the resolved object.
+You can prefix the value with a `/` for making it relative to the site root,
+otherwise it is relative to the item it is defined in ("Foo" in the above
+example).
```

Comment:
The above example has a prefixing `/` - so it is relative to the site root, not to "Foo", right? So this text is wrong.